lighthouse-core/runner.js

```diff
@@ -33,6 +34,8 @@ class Runner {
 
     const config = opts.config;
 
+    Sentry.init(opts.flags.disableErrorReporting === false, opts.environmentData || {});
```

Comment:
is having ` || {}` going to be confusing in the data? runner may want to try to infer more to differentiate different environments (but you might already be planning that)